### PR TITLE
Remove unused contracts

### PR DESCRIPTION
**Pull request by skalermo**

### CODE REVIEW

**Comment by yuchenlintt** [comment]
Thanks for taking on this task! Culling zombie code is one of the best things for repo health.

I can't comment on all the deletions, but I know:
- CrvPriceOracle is still used in our CurveYearnStrategy
- SushiTimelock and TruSushiswapRewarder are still used for the Sushiswap farm

Could you split this PR into smaller PRs so it's easier to evaluate on a contract-by-contract basis what we can safely delete?

**Comment by barrutko** [comment]
I agree with @yuchenlintt. Let's issue multiple smaller PRs, one by one, with a smaller number of removals. For example, let's start with extracting the removal of Avalanche stuff to separate PR.

File: contracts/avalanche/AvalancheTokenController.sol [hunk 1]
```diff
@@ -1,610 +0,0 @@
-// SPDX-License-Identifier: MIT
```
Comment (hal909) [inline contracts/avalanche/AvalancheTokenController.sol:1]: Keep this file. We still need to maintain Avalanche smart contracts

File: contracts/avalanche/AvalancheTrueUSD.sol [hunk 2]
```diff
@@ -1,36 +0,0 @@
-// SPDX-License-Identifier: MIT
```
Comment (hal909) [inline contracts/avalanche/AvalancheTrueUSD.sol:1]: Keep this file.

File: contracts/truefi/SushiTimelock.sol [hunk 3]
```diff
@@ -1,142 +0,0 @@
-// SPDX-License-Identifier: MIT
```
Comment (hal909) [inline contracts/truefi/SushiTimelock.sol:1]: Keep this file.

File: contracts/truefi/TruSushiswapRewarder.sol [hunk 4]
```diff
@@ -1,89 +0,0 @@
-// SPDX-License-Identifier: MIT
```
Comment (hal909) [inline contracts/truefi/TruSushiswapRewarder.sol:1]: Keep this file

**Review by hal909 — CHANGES_REQUESTED** [review]
There are some files we need to keep.
1. AvalanceTrueUSD
2. AvalancheTokenController
3. SushiTimeLock
4. SushiSwapRewarder

**Comment by skalermo** [comment]
Removal of the unused contracts was split into separate PRs #763, #764 and #765 and merged. As for the Avalanche contracts (#763) we will have to restore them. 
I think we can close this PR.

**Comment by yuchenlintt** [comment]
Apologies about the miscommunication, Hal. We probably should have closed this PR once we split it, to avoid duplicate review work. Closing this now and reverting #763.

**Pull request closed without merging**